src/OpenTelemetry/Metrics/Exemplar/Exemplar.cs

```diff
@@ -20,39 +19,150 @@ namespace OpenTelemetry.Metrics;
 #endif
 public
 #else
-/// <summary>
-/// Represents an Exemplar data.
-/// </summary>
-#pragma warning disable SA1623 // The property's documentation summary text should begin with: `Gets or sets`
 internal
 #endif
     struct Exemplar
 {
+    internal HashSet<string>? ViewDefinedTagKeys;
```

Comment:
Suggestion for another PR: Both Exemplars and MetricPoint updates could benefit from using `FrozenSet` instead of `HashSet` on .NET 8.

Reply:
I had the same thought when I was working on this!